src/hotspot/cpu/x86/matcher_x86.hpp

```diff
@@ -188,6 +188,10 @@
     switch(vopc) {
       default:
         return 0;
+      case Op_VectorCastF2X: // fall through
+      case Op_VectorCastD2X: {
+        return is_subword_type(ety) ? 75 : 70;
```

Comment:
Add comment here and in other cases explaining numbers. Is it size of instructions or elements or something?

Reply:
Value now matches the one for RoundV[FD] IR nodes, currently, its a rudimentary heuristic based on emitted code size for complex IR nodes used by unroll policy.

Reply:
Please, add this information/clarification to this method's comment at line 186.